pinax/stripe/actions/customers.py

```diff
@@ -43,27 +43,21 @@ def create(user, card=None, plan=settings.PINAX_STRIPE_DEFAULT_PLAN, charge_imme
     Returns:
         the pinax.stripe.models.Customer object that was created
     """
-    trial_end = hooks.hookset.trial_period(user, plan)
-    stripe_customer = stripe.Customer.create(
-        email=user.email,
-        source=card,
-        plan=plan,
-        quantity=quantity,
-        trial_end=trial_end
-    )
-    cus, created = models.Customer.objects.get_or_create(
-        user=user,
-        defaults={
-            "stripe_id": stripe_customer["id"]
-        }
-    )
+    cus, created = models.Customer.objects.get_or_create(user=user)
```

Comment:
StripeObject.stripe_id can't be null at present, so would this not fail if it came to creating?

Reply:
It is a CharField, and can be blank (validation for the default `blank=False` is skipped here).
But there can only be one blank `stripe_id` (it is a unique) field.